app/validations/household_validations.rb

```diff
@@ -56,6 +51,21 @@ def validate_household_pregnancy(record)
     end
   end
 
+  def validate_household_number_of_other_members(record)
```

Comment:
Shall we refactor that before this goes in to make that clearer?

Reply:
sure, if changing the pattern isn't too difficult

Reply:
How do you both think would be the best way to refactor?

Reply:
Actually I think it makes sense to consider that out of scope for this PR since this is how it's already working. Can do that as a separate thing if needed?